core/sawtooth/client.py

```diff
@@ -54,8 +54,8 @@ def _sign_message_with_transaction(transaction, message_type, key):
     of a sha256 hexdigest.
     """
     transaction['Nonce'] = time.time()
```

Comment:
Are these 0.7-style transactions?  Is this code still necessary?  Maybe not an item for this PR, but something we should be aware of.